src/pages/index.tsx

```diff
@@ -82,30 +87,32 @@ export default function Page() {
         </Text>
         <FrameworkGrid currentKey="javascript" />
       </Flex>
-      {/* @Todo: Add href links for remaining cards, tracking this with separate
-      task */}
       <LinkCardCollection>
-        <LinkCard
-          isExternal={true}
-          href={''}
-          icon={() => <IconGithub fontSize="2rem" />}
-        >
-          JavaScript Libraries on Github
-        </LinkCard>
+        {linkCardPlatformData ? (
+          <LinkCard
+            isExternal={true}
+            href={''}
```

Comment:
Would it be possible set up the value we need to pass into the `href` here? Like how you have `{linkCardPlatformdata.roadmap}` in some other spots

Reply:
Sorry ,updated @timngyn